examples/fluids/problems/stg_shur14.c

```diff
@@ -371,6 +372,10 @@ PetscErrorCode SetupStrongSTG(DM dm, SimpleBC bc, ProblemData *problem, Physics
       // {1,2,3,4} for u, v, w, T
       for (int i = 0; i < 4; i++) comps[i] = i + 1;
       break;
+
+    case STATEVAR_ENTROPY:
+      for (int i = 0; i < 5; i++) comps[i] = i;
+      break;
```

Comment:
I'm also not sure about this. Setting `comps` to set all values at the boundary would over-constrain the problem setup (this is used to set the components strong boundary conditions). I'm currently refreshing my memory of the strong BC setup right now to see what this should be realistically.

Reply:
I think it should be the last 4 components of the entropy variables, since those involve $T$ and $\mathbf{u}$.